src/control.py

```diff
@@ -1,35 +1,96 @@
-#Python script to read analog voltage values from ADS1115 16bit ADC with PGA
-#Author: Carson Berry 
-#Date: February 2nd, 2021
-#usage: 
-#    python3
-#    import src.adc as adc
-#    sensors = adc.adc_sensors()
-#    sensors.read_leak()
-#    sensors.read_pH()
-# Reference provided at: https://learn.adafruit.com/adafruit-4-channel-adc-breakouts/python-circuitpython
+
+#----------------------------------------------Template--------------------------------------------------#
```

Comment:
I like this Template description! Would you please be able to add all of the example code together at the end of it?

Reply:
Sure